README.md

```diff
@@ -63,6 +63,8 @@ class TopicsController < ApplicationController
 end
 ```
 
+The default behavior is to [return 200](https://github.com/markets/invisible_captcha/blob/c60e35a140144df41c50468dc2e6291716868040/lib/invisible_captcha/controller_ext.rb#L41).
```

Comment:
hey @jjb 👋🏼 

This is already explained in the previous paragraph, so I'd probably just add the `n.b` note here.

Thanks for contributing!

Reply:
whoops, so it it! done.